app/models/grant_submission/response.rb

```diff
@@ -153,19 +158,19 @@ def validate_length_if_short_text_response
     end
 
     def validate_attachment_size_if_file_upload_response
-      if document.blob.byte_size > MAXIMUM_DOCUMENT_FILE_SIZE
+      if self.document.blob.byte_size > MAXIMUM_DOCUMENT_FILE_SIZE
         errors.add(:document, :file_too_large, question: question.text,
                                                allowed_file_size: READABLE_MAXIMUM_DOCUMENT_FILE_SIZE,
-                                               uploaded_file_size: (document.byte_size.to_f/(1.megabyte)).round(1))
-        document.purge
+                                               uploaded_file_size: (self.document.byte_size.to_f/(1.megabyte)).round(1))
+        document = nil
```

Comment:
I seem to recall from talking to @fanglinnw that this is not needed.

Reply:
yes, this is not needed